README.md

```diff
@@ -70,10 +70,12 @@ make smoketest
 These require the [APIGEE get_token tool](https://docs.apigee.com/api-platform/system-administration/using-gettoken) and a correctly configured environment:
 
 ```
+SSO_LOGIN_URL=https://login.apigee.com
```

Comment:
this is a repeat of line 77 - I think this line is correct and you need to remove line 77 (since this line is required for `get_token`)